pkg/k8sutil/context.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package k8sutil
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Context struct {
```

Comment:
This struct will have other types of contexts such as for the CLI. While there may be a better package name for this file, it doesn't seem worth creating a package just for that. Perhaps it will be clear in the future if there is a better package as more functionality is added.

Reply:
I saw in rook we have a different package so I followed that but I'm happy to revert too, let me know.